### PR TITLE
Reformat the whole QC code (QC-100)

### PR DESCRIPTION
**Pull request by Barthelemy**
QC-100

### CODE REVIEW

File: Modules/Skeleton/src/SkeletonCheck.cxx [hunk 1]
```diff
@@ -6,75 +6,68 @@
...
 
-namespace o2 {
-namespace quality_control_modules {
-namespace skeleton {
-
-SkeletonCheck::SkeletonCheck()
+  namespace o2
 {
-}
+  namespace quality_control_modules
```
Comment (knopers8) [inline Modules/Skeleton/src/SkeletonCheck.cxx]: something bad happened with the namespace indents

File: Framework/include/QualityControl/TaskDataProcessorFactory.h [hunk 2]
```diff
@@ -8,9 +8,12 @@
...
 
-namespace o2 {
-namespace quality_control {
-namespace core {
+namespace o2
+{
+namespace quality_control
+{
+namespace core
+{
```
Comment (knopers8) [inline Framework/include/QualityControl/TaskDataProcessorFactory.h]: How about changing nested namespaces to c++17 syntax - `namespace o2::quality_control::core {`? Not now, but when we have common agreement on that.

Comment (Barthelemy) [reply Framework/include/QualityControl/TaskDataProcessorFactory.h]: I think that it is a good idea. I was not aware of it. Could you create a ticket in JIRA ? 
O2 General

**Comment by Barthelemy** [comment]
@knopers8 I have fixed the formatting

**Pull request merged**